storage/libradb/src/event_store/mod.rs

```diff
@@ -114,7 +114,6 @@ impl EventStore {
                 // Queries tend to base on very recent ledger infos, so first try to linear search
                 // from the most recent end, for limited tries.
                 // TODO: Optimize: Physical store use reverse order.
-                let mut n_try_recent = 10;
```

Comment:
Why did you remove this line? seems not related to this PR.

This breaks `cargo test -p consensus`

Reply:
Hm, didn't notice the implication, it's from automation tool fix, will submit a revert PR for this line. Thanks!

Reply:
Thanks for confirming, It's fixed in https://github.com/libra/libra/pull/543